package.json

```diff
@@ -1,9 +1,20 @@
 {
-  "name": "tc39-web-draft",
-  "private": true,
+  "name": "@tc39/tc39.github.io",
   "version": "0.0.0",
-  "description": "Draft for TC39 website",
-  "main": "index.js",
+  "description": "Get involved in specifying JavaScript",
+  "homepage": "https://github.com/tc39/tc39.github.io",
+  "bugs": {
+    "url": "https://github.com/tc39/tc39.github.io/issues"
+  },
```

Comment:
all of these fields are only used when a package is published, so there's no need for them to be here

Reply:
Yeah I’m not sure if all of these are needed.

I would change or remove `maintainers` as I believe ujjwal, myself and potentially others are still around too. Yulia also created this site with myself, Ujjwal and some others initially too so having the concept of a single “author” doesn’t make much sense in this context.

node we can keep for local development, license is fine, description and name are ok to keep too

Reply:
Those opinions certainly sound reasonable. I reflected it in the latest commit.